tests/test_expectation_maximization.py

```diff
@@ -33,13 +34,13 @@ def test_split_array(test_ir, n_pix):
 def test_build_ctf_array(test_ir):
     """Test bulding of arbitrary CTF array."""
     ctfs = test_ir.build_ctf_array()
-    assert ctfs.shape == (0, 0, 0)
+    assert ctfs.shape == (n_pix, n_pix, n_pix)
```

Comment:
Should be `(n_ctfs, n_pix, n_pix)`.

Write the tests so the shapes make sense for the design of the code.